src/utils/filter-data.ts

```diff
@@ -22,7 +22,7 @@ type FilterMetadataByOption<T extends readonly string[]> = Record<
 /**
  * filter constants
  */
-export const PLATFORM_FILTER_OPTIONS = ["js", "android", "ios", "flutter"];
+export const PLATFORM_FILTER_OPTIONS = ["android", "js", "ios", "flutter", "react-native"];
```

Comment:
`react-native` is in `FRAMEWORK_FILTER_OPTIONS` so this results in duplications in `INTEGRATION_FILTER_OPTIONS`.  Generally all keywords up to now have fit neatly into either (web) frameworks (`/ui`?) or platforms (`/lib`, `/guides`) but it seems like the goal of this change is to make React Native a member of both.  This can have a lot of effects, for example I would expect RN to appear twice on the getting-started pages' dropdown.  Can we have a preview docs site for this @timngyn?

Reply:
I guess it wouldn't affect the dropdown since that just reads from the directory list, but we should still look to make sure that this change doesn't have any accidental effects.

Reply:
I tried to double check to make sure it doesn't cause any other issues. But will leave it up to @timngyn to see if this should be done differently.

Reply:
there's a few edge cases that are slowly coming back to me now that I'm looking at the codebase again 😅  for example, ChooseFilterPages like [/start/](https://docs.amplify.aws/start/) (with cookies cleared/in incognito) that I would want to just check briefly if we had a preview site up.  if we do find an issue there just something like `[...new Set([...PLATFORM_FILTER_OPTIONS, ...FRAMEWORK_FILTER_OPTIONS])]` instead of the currently existing
```
const INTEGRATION_FILTER_OPTIONS = [
  ...PLATFORM_FILTER_OPTIONS,
  ...FRAMEWORK_FILTER_OPTIONS,
];
```
would certainly work.  hopefully I'm just being paranoid over nothing here 😄